tools/model-compiler/src/main/java/io/spine/tools/compiler/enrichment/ByOption.java

```diff
@@ -38,18 +38,13 @@
 
 /**
  * Obtains event names from the {@code "by"} field option of a message.
- *
- * @author Alexander Litus
- * @author Alexander Yevsyukov
  */
-class ByOption implements Logging {
+final class ByOption implements Logging {
 
-    private final String packagePrefix;
     private final DescriptorProto message;
     private final FieldDescriptorProto field;
 
-    ByOption(String packagePrefix, DescriptorProto message, FieldDescriptorProto field) {
-        this.packagePrefix = packagePrefix;
+    ByOption(DescriptorProto message, FieldDescriptorProto field) {
```

Comment:
Please check if we're able to migrate to `MessageType` instead of two descriptors.

Reply:
The second descriptor is one for a field. Probably `ByOption` would get more tricks and be based on `MessageType` in another PR related to enrichments.